project1/css/index.css

```diff
@@ -54,11 +54,92 @@ table {
 
 html, body {
     height: 100%;
-    font-family: 'Titillium Web', sans-serif;
+	font-family: 'Titillium Web', sans-serif;
 }
 
 h1, h2, h3, h4, h5 {
     font-family: 'Bangers', cursive;
     letter-spacing: 1px;
     margin-bottom: 10px;
 }
+body{
+	display: flex;
+	justify-content: center;
+}
+.container {
+  width: 800px;
+}
+
+header {
+  display: flex;
+  width: 100%;
+  
+}
+
+header nav ul {
+  display: flex;
+  list-style: none;
+  justify-content: space-around;
+}
+
+header nav {
+  
+  width: 600px;
+  flex-grow: 1
+}
+
+header nav ul li a {
+  text-decoration: none;
+  color: grey;
+}
+
+.logo {
+justify-content: flex-end;
```

Comment:
This code block is actually unnecessary and can be omitted.

Also missing indentation.